include/wx/html/htmlcell.h

```diff
@@ -115,8 +115,15 @@ class WXDLLIMPEXP_HTML wxHtmlRenderingStyle
 class WXDLLIMPEXP_HTML wxDefaultHtmlRenderingStyle : public wxHtmlRenderingStyle
 {
 public:
+    explicit wxDefaultHtmlRenderingStyle(wxWindowBase const* wnd = 0)
+        : m_wnd(wnd)
+    {}
+
     virtual wxColour GetSelectedTextColour(const wxColour& clr) wxOVERRIDE;
     virtual wxColour GetSelectedTextBgColour(const wxColour& clr) wxOVERRIDE;
+
+private:
+    wxWindowBase const* const m_wnd;
```

Comment:
```suggestion
    const wxWindowBase* const m_wnd;
```

Also need `wxDECLARE_NO_COPY_CLASS()` here, if only to suppress warnings from some compilers about them being unable to generate copy ctor.